example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
-    - YOUR_CANISTER_ID  # propose_sns.sh can fill this in automatically.
-    # More canisters can be listed here.
+    # You need to replace this for your own SNS decentralization swap.
+    # For testing, propose_sns.sh will fill this in automatically.
+    - YOUR_CANISTER_ID
 
+# Configuration of SNS tokens in the ICRC Ledger canister deployed as part
+# of the SNS.
 Token:
+    # The name of the token issued by the SNS ledger.
+    # Must be a string with a length between 4 and 255 characters.
+    #
+    # Example: InternetComputerProtocol
     name: Rock Out Token
+
+    # The symbol of the token issued by the SNS Ledger.
+    # Must be a string with a length between 3 and 10 characters.
+    #
+    # Example: ICP
     symbol: ROT
-    transaction_fee: 10_000 e8s # You can leave this alone.
 
-# It is fine to leave these alone.
+    # SNS ledger transactions will have this fee.
+    transaction_fee: 10_000 e8s
+
+    # Path to the SNS token logo on your local filesystem. The path is relative
+    # to the configuration file location, unless an absolute path is given.
+    # Must have less than 341,334 characters, roughly 256 Kb. The only supported
+    # format is PNG.
+    logo: logo.png
+
+# Configures SNS proposal-related fields. These fields have no
+# effect on the NNS proposal that is created from this file, but will affect all
+# SNS proposals submitted to the newly created SNS.
 Proposals:
+    # The cost of making an SNS proposal that is rejected by the SNS Neuron
+    # holders.
     rejection_fee: 1 token
+
+    # The initial voting period of a newly-created SNS proposal. A proposal's
+    # voting period may be increased during its lifecycle due to the
+    # wait-for-quiet algorithm (see details below).
     initial_voting_period: 4d
+
+    # The wait for quiet algorithm extends the voting period of a proposal when
+    # there is a flip in the majority vote during the proposal's voting period.
+    #
+    # Without this, there could be an incentive to vote right at the end of a
+    # proposal's voting period, in order to reduce the chance that people will
+    # see and have time to react to that.
+    #
+    # If this value is set to 1 day, then a change in the majority vote at the
+    # end of a proposal's original voting period results in an extension of the
+    # voting period by an additional day. Another change at the end of the
+    # extended period will cause the voting period to be extended by another
+    # half-day, etc.
+    #
+    # The total extension to the voting period will never be more than twice
+    # this value.
+    #
+    # For more information, please refer to
+    # https://wiki.internetcomputer.org/wiki/Network_Nervous_System#Proposal_decision_and_wait-for-quiet
     maximum_wait_for_quiet_deadline_extension: 1 day
 
-# It is fine to leave these alone.
+# Configuration of SNS neuron-related fields. These fields have no effect on the
+# NNS proposal, but will affect the SNS itself.
```

Comment:
```suggestion
# SNS nervous system parameters related to neurons. These fields have no effect on the
# NNS proposal, but will affect the SNS itself.
```

Reply:
I suggest that we don't assume that the users of this file have read the code.

So here's another option:

```suggestion
# Configuration of SNS neurons. These fields have no effect on the NNS proposal,
# but will affect the SNS itself.
```

Reply:
Do you mean this is why you wouldn't call them "nervous system parameters"?
I think this is also the name that we should adopt even for users who haven't read the code - for example the proposal's name is also like that and this is what we call them in the documentation (so my suggestion of the name is not like that because this is the variable name)